<!--RGBD Feature viewer -->

<launch>

  # ORB, SURF, GTF, STAR
  <arg name="detector_type" default="GFT"/> 

  <node pkg="ccny_rgbd" type="feature_viewer_node" name="feature_viewer_node" 
    output="screen">
      
    #### features #####################################
    
    #  ORB, SURF, or GFT (Good features to track)
    <param name="feature/detector_type"       value="$(arg detector_type)"/> 
    <param name="feature/smooth"              value="0"/>
    <param name="feature/max_range"           value="7.0"/>
    <param name="feature/max_stdev"           value="0.03"/>
    <param name="feature/show_keypoints"      value="false"/>
    <param name="feature/publish_cloud"       value="false"/>
    <param name="feature/publish_covariances" value="false"/>

    <!-- you can use dynamic reconfigure for the parameters above, as well as
    specific feature type parameters-->
    
  </node>
  
  <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find ccny_rgbd)/launch/feature_viewer.vcg"/>
  
  <node pkg="dynamic_reconfigure" type="reconfigure_gui" name="reconfigure_gui"/>
  

</launch>

